notes/lecture5.tex

```diff
@@ -0,0 +1,141 @@
+\section{Lecture 5: Conditional Gradient (Frank-Wolfe)}
+\sectionlabel{frankwolfe}
+In this lecture we discussed about the conditional gradient method, also known as the Frank-Wolfe (FW) algorithm. The motivation of using this approach is the projected gradient descent can be computationally inefficient under certain scenarios.
+
+\subsection{Intuition behind Frank-Wolfe algorithm}
+We motivate this lecture by the computational inefficiency that projected gradient can have. With conditional gradient, we are able to sidestep some of these inefficiencies. An intuitive idea of the FW algorithm is as follows. \\
+\\We start from $x_0$. Then, for t = 1 to T steps, we set
+$$ x_{t+1} = x_t + \eta_t(\bar{x_t}-x_t) $$
+where
+$$ \bar{x_t} = \underset{x \in \domain}{\operatorname{argmin}} f(x_t) + \nabla f(x_t) ^\trans (x-x_t)$$
+Since we hope to minimize with respect to $x \in \domain$, we can simplify the equation.
+$$ \bar{x_t} = \underset{x \in \domain}{\operatorname{argmin}} \nabla f(x_t) ^\trans x $$
+Note that we need step size $\eta_t \in [0,1]$ to guarantee $x_{t+1} \in \domain$.
+
+\subsection{Theorem: conditional gradient convergence analysis}
+\begin{theorem}[Convergence Analysis]
+\theoremlabel{convergence}
+Assume:
+\begin{itemize}
+\item $f\colon\domain\to\R$ is convex and $\beta$-smooth
+\item $\exists$ an optimal point, $x^*$, such that $x^* \in\domain$ and $\nabla f(x^*)$ = 0
```

Comment:
These assumptions don't read properly. You can't start a sentence with `\exists`. Also don't start sentences with symbols in general.